pages/ksphere/kommander/1.1/release-notes/index.md

```diff
@@ -50,369 +50,109 @@ Template:
 
 -->
 
-### Kommander 1.1.0 RC 7 - July 16th 2020
+#### New Features
+
+##### Centralized Cost Monitoring
+
+Kubecost, running on Kommander, provides centralized cost monitoring for all managed clusters. This feature, installed by default in every Kommander cluster, provides a centralized view of Kubernetes resources used on all managed clusters. For more information go to [Centralized Cost Monitoring](https://docs.d2iq.com/ksphere/kommander/1.1.0/centralized-cost-monitoring/)
+
+##### D2-ify Managed Clusters
+
+When attaching non-Konvoy clusters, such as Amazon EKS, Azure AKS, Google GKE, and On-Premises Kubernetes clusters, Kommander will federate a subset of standard Konvoy Addons to enable SSO, AuthN/Z, observability, and cost monitoring. This includes addons: Prometheus Operator, Alert Manager, Karma, Elasticsearch, Kibana, and Fluentbit.
+
+##### Improved RBAC
+
+Kommander has revamped Access Controls for users at the global, workspace, and project levels allowing for greater flexibility and security when assigning roles. See [Granting Access to Kubernetes Resources in Kommander](https://docs.d2iq.com/ksphere/kommander/1.1.0/tutorials/configure-rbac/)
+
+##### LDAP Support
+
+Kommander now supports configuring an external LDAP directory for authenticating users. See [Connect and configure your LDAP directory to Konvoy and Kommander](https://docs.d2iq.com/ksphere/kommander/1.1.0/ldap/setup-ldap/)
```

Comment:
Not 100% sure if this was supported before but saw new docs for it

Reply:
thats old. but got some improvements IIRC. But SAML was added AFAIK.

Reply:
I updated that topic based on Docathon input. It a long topic but hopefully it covers everything.

Reply:
hm... there is literally not a single mention of SAML in all the kommander docs. And from this doc https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/operations/identity-providers/

```
Currently, Kommander supports Github, LDAP, and any standard OIDC provider such as Google.
```

Lucy you got some 'splaining to do

Anyway if LDAP is old I'll take it out from the new features section

Reply:
yeah… was referring to https://jira.d2iq.com/browse/D2IQ-65025, but it didnt made it into the UI 🙈  we got a JIRA to add it and will update docs once thats done.

Reply:
so yeah, removing it for this release is probably correct.